src/functionalTest/resources/features/F-001/F-001.feature

```diff
@@ -0,0 +1,137 @@
+@F-001
+Feature: F-001: Retrieval of metadata from doc-store for given document id
+
```

Comment:
I would review the test data underlying this feature file but I'm  little confused: The PR had been merged but the Feature file is not available in development or master branch. It looks like it was deleted after the merge of this PR. It will be more convenient to discuss interactively.